draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1200,7 +1201,7 @@ Error Handling:
 
 ## Monotonicity and Counters {#monotonicity_counters}
 
-Monotonicity is the backbone of time-based sortable UUIDs. Normally, time-based
+Monotonicity (each subsequent value being greater than the last) is the backbone of time-based sortable UUIDs. Normally, time-based
```

Comment:
I think providing the inline definition of "monotonicity" here is helpful, since strictly speaking from the mathematical definition this could also describe values that count down, or that count in a constant direction based on specific input.  We might as well spell out exactly what we mean here.